### ISSUE TITLE
Accordion

### ISSUE THREAD

**Issue by mattkerlogue** [ukgovdatascience/govdown#35]
As and when there's time it'd be great to support the [accordion](https://design-system.service.gov.uk/components/accordion/) component.

**Comment by nacnudus** [comment]
Similar syntax to tabsets? But without the extra header for narrow screens.

```markdown
::: {.tabset}
# Header appears on narrow screens

## Tab header 1

Tab content A

## Tab header 2

Tab content B

Second paragraph of content.
:::

**Comment by mattkerlogue** [comment]
Yeah I think that would be best - would make it easy to switch between the two. So it'd be as follows:

``` r
::: {.accordion}

## Accordion 1

Accordion content 1.

## Accordion 2

Accordion content 2.

Second paragraph.
:::
```

**Comment by nacnudus** [comment]
Closed in cf7d583c2e8332668a43c03b248f67a9018a7809
